src/AI/mcp/server/tools/layout_components_tool/layout_components_tool.py

```
@@ -30,11 +30,29 @@
+description="""
+This tool helps you choose and configure **Altinn Studio layout components** at a generic level.
+
+Use this tool when you want to:
+- Discover which UI components exist in Altinn Studio (e.g. Header, Input, Date picker, Dropdown, etc.)
+- Understand what properties, textResourceBindings, dataModelBindings, and expressions a component supports
+- Get example configurations for a component you are planning to use
+
+**Important limitations**
+
+- This tool **does not read, search, or inspect your specific application**.
+- It **cannot locate components by id** (for example `'/header-title'`) or return the exact configuration from your app’s layouts.
```

Ruff (0.15.1)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)